modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/processors/SW360EnricherImpl.java

```diff
@@ -73,12 +76,15 @@ private void mapExternalIdsOnSW360Release(SW360Release sw360Release, Artifact ar
         mapCoordinates(sw360Release)
                 .forEach(artifact::addFact);
 
-        artifact.addFact(new DeclaredLicenseInformation(makeLicenseStatementFromString(sw360Release.getDeclaredLicense())));
-        artifact.addFact(new ObservedLicenseInformation(makeLicenseStatementFromString(sw360Release.getObservedLicense())));
+        addLicenseFact(Optional.ofNullable(sw360Release.getDeclaredLicense()), artifact, DeclaredLicenseInformation.class);
```

Comment:
The function `makeLicenseStatementFromString` is now unused

Reply:
That is not correct, it is used in the addLicenseFact method in a map statement.